include/mbgl/util/tuple.hpp

```diff
@@ -2,18 +2,20 @@
 
 // Polyfill needed by Windows because MSVC STL
 // is not compatible with our IndexedTuple code
-#if defined(_WINDOWS)
+#if 0
```

Comment:
We should remove the polyfill from the code and `tao` from cmake if it is no longer needed.